src/analytics.ts

```diff
@@ -1,5 +1,6 @@
 export const trackEvent =
-  process.env.REACT_APP_GOOGLE_ANALYTICS_ID &&
+  typeof process !== "undefined" &&
+  process?.env?.REACT_APP_GOOGLE_ANALYTICS_ID &&
```

Comment:
if we have `?` on `process` do we need the `typeof`?

Reply:
Optional chaining will not work for an undeclared object so for this case it wont work

Reply:
then we can remove the  first `?`

Reply:
```suggestion
  process.env?.REACT_APP_GOOGLE_ANALYTICS_ID &&
```